Source_Code/Client/ClueLessClient.java

```diff
@@ -319,38 +351,60 @@ else if( statUp instanceof TurnUpdate)
             if(((TurnUpdate) statUp).activeTurn )
             {
                 UserPlayer.PlayerTurn = true; // Set the turn status to true
-                System.out.println( "[Server] It is now your turn.");
-                System.out.println("\n****Enter a command****");
+                System.out.println( "\n[Server] It is now your turn.");
+                System.out.println("\n\n****Enter a command****");
                 System.out.println("Move Left: 1");
                 System.out.println("Move Right: 2");
                 System.out.println("Move Up: 3");  // To test sending multiple
                 System.out.println("Move Down: 4");
-                System.out.println("Exit: -1\n");
-                System.out.println( UserPlayer.getAllCardsString() );
-                System.out.print("\n>> ");
+                System.out.println("Suggest: 5");
+                System.out.println("Exit: -1");
             }
-            
-            
-            
+
+
+
             else
             {
                UserPlayer.PlayerTurn = false; // Set the turn status to false
                System.out.println( "[Server] Ended turn.");
             }
-            
+
         }
-        
-        
+
+
         else if( statUp instanceof PlayerHandUpdate)
         {
-           UserPlayer.setHand( ((PlayerHandUpdate) statUp).getHandUpdate() );
-           System.out.println( UserPlayer.getAllCardsString() + "\n");
-           //System.out.println( ((PlayerHandUpdate) statUp).getHandUpdate() + "\n");
-            
+            //UserPlayer.setHand( ((PlayerHandUpdate) statUp).getHandUpdate() );
+            //System.out.println( UserPlayer.getAllCardsString() + "\n");
+           System.out.println( ((PlayerHandUpdate) statUp).getHandUpdate() + "\n");
```

Comment:
I think this was also changed in David's most recent commit? If so, should be changed back